index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Apple</title>
+        <link rel="stylesheet" href="styles/styles.css">
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        
+        <style>
+            @import url('https://fonts.googleapis.com/css?family=Roboto:100');
+        </style>
+    </head>
+    <body> 
+        <div class="wrapp-all">
+          <nav>
+              <header class="elements first-item">       
+                  <ul>
+                      <li><img src="images/logo.png"></li>
+                      <li><span>Store</span></li>
+                      <li><span>Mac</span></li>
+                      <li><span>iPod</span></li>
+                      <li><span>iPhone</span></li>
+                      <li><span>iTunes</span></li>
+                      <li><span>iPad</span></li>
+                      <li><span>Support</span></li>
+                      <li><input type="search" name="search"></li>
+                    </ul>    
+                      
+              </header>
+          </nav>
+
+            <main>   
+                <div class="link1">
+                    <a href=""><h1>What will your verce be?</h1></a><br />
+                </div>  
+
+                <div class="link2">
+                    <a href=""><h2>Everyone has something to share. See how people are using <br />
+                      iPad in their unique ways to add to the world's story.</h2></a> 
+                </div>
+                
+                <div class="link3">
+                    <a href="playVideo"><h3>Watch the "Your Verce" film and explore the stories</h3></a> 
+                </div>
+                                                                                                   
+            </main>
+
+        <footer>
+            <div class="pictures" style="display: flex">
+              <div class="links l2">
```

Comment:
It's against best practices to use inline styling as it is harder to debug and also harder to change in the future.